nexustreetojson.py

```diff
@@ -44,16 +86,22 @@ def _tree_to_json_string(tree):
             names = sorted(entries)
             for k in names:
                 children.append(_tree_to_json_string(entries[k]))
-            result.append('"children": [' + ','.join(children) + ']')
+            result.append(children_prepend + '"children": [' + ','.join(children) + ']' + children_append)
```

Comment:
Do you build up the json return value via string concatenation because the order is important?